components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/internal/SuggestionGroup.kt

```diff
@@ -23,6 +25,11 @@ internal fun SuggestionGroup(
         title,
         color = colors.groupTitle,
         modifier = Modifier
-            .padding(4.dp)
+            .padding(
```

Comment:
Is it possible for this to be dynamic? In case the application side did not pass in a group title?

Reply:
SuggestionGroup composable is never called if the group title is empty. See below

`group.title?.let { title ->
                 tem(group.id) {
                        SuggestionGroup(title, colors)
`